packages/mobile/src/transactions/ExchangeFeedItem.tsx

```diff
@@ -36,6 +46,10 @@ export function ExchangeFeedItem(props: Props) {
   const outCurrency = resolveCurrency(outSymbol)
   const dollarAmount = inCurrency === CURRENCY_ENUM.DOLLAR ? inValue : outValue
```

Comment:
A lot of repeated ternary conditions here. Consider reorganizing this similar to what I did here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/exchange/ExchangeTradeScreen.tsx#L188